test/functional/wallet_address_types.py

```diff
@@ -364,12 +364,12 @@ def run_test(self):
 
         if self.options.descriptors:
             self.log.info("Nodes with addresstype=bech32 always use either a bech32 or bech32m change output (unless changetype is set otherwise):")
-            self.test_change_output_type(3, [to_address_bech32_1], 'bech32m')
-            self.test_change_output_type(3, [to_address_p2sh], 'bech32')
+            self.test_change_output_type(3, [to_address_bech32_1], 'bech32')
+            self.test_change_output_type(3, [to_address_p2sh], 'p2sh-segwit')
         else:
             self.log.info("Nodes with addresstype=bech32 always use a P2WPKH change output (unless changetype is set otherwise):")
             self.test_change_output_type(3, [to_address_bech32_1], 'bech32')
-            self.test_change_output_type(3, [to_address_p2sh], 'bech32')
+            self.test_change_output_type(3, [to_address_p2sh], 'p2sh-segwit')
```

Comment:
The log comment above this lines aren't valid anymore.

Reply:
thx, done